data/legal.txt

```diff
@@ -1,9 +1,8 @@
-Copyright 2014 Ball Aerospace & Technologies Corp.
+Copyright 2014 Ball Aerospace
```

Comment:
Ball Aerospace & Technologies Corp. is still the companies legal name.   Ball Aerospace is more of a branding / Doing Business As name.